esprima.js

```diff
@@ -2190,6 +2269,13 @@
         return lookahead.type === Token.Keyword && lookahead.value === keyword;
     }
 
+    // Return true if the next token matches the specified contextual keyword
+    // (where an identifier is sometimes a keyword depending on the context)
+
```

Comment:
you can expand the diff, I'm using the existing pattern from similar methods (empty line before and after the method definition...